### DISCUSSION TITLE
How to update?

### DISCUSSION THREAD

**Discussion by puzkos** [givanz/Vvveb#33] [General]
Hi 

Best product, but... it`s very alpha versions... How can we discouse about cooperations?)

And how to update to 0.0.3 ver? the button does not work...(

Best
Serhii

**Comment by givanz** [comment]
Hi

Thank you.

The project is not finished yet, there are still bugs that are being fixed, the code is provided for preview and testing only,  a fully working beta will be released soon. 

If you want to discuss about the project you can use the email address from my profile page.

The release didn't have the version increased to 0.0.3 automatically and after the update it still said 0.0.2 this is the reason it looked that it didn't work, it's fixed now.

**Reply by puzkos** [reply]
Hi

Yes, now it`s 0.0.3
Thank you!

Yes, i understend, that it`s alpha and have a lot of bugs... 
But it`s amaizing! :)

I`m texting to your email..

Also after update all plugins were disabled.

Also I founded a lot of other bugs, I will text about all of them tommorrow, after I will test new version/

**Reply by givanz** [reply]
> Also after update all plugins were disabled.

Sorry about this, the reason is that the update overwrote the plugin configuration, I changed the update process to keep the active plugin list configuration, this will not happen on future updates.

**Comment by puzkos** [comment]
![image](https://github.com/givanz/Vvveb/assets/97404154/5b7ed6d8-01bb-46e2-bff4-b57ef86e48e0)


Before the update, payment and delivery methods were displayed here.
Before the update, the order was not placed.

After a successful update to 0.0.3, all modules were disabled, I re-enabled them. But now they don't show up here.

When the page is refreshed, the number of product orders increases by 1.
When you click on the order button in the screenshot, nothing happens.

Best
Serhii

**Reply by givanz** [reply]
> Before the update, payment and delivery methods were displayed here.

> After a successful update to 0.0.3, all modules were disabled, I re-enabled them. But now they don't show up here.

This is probably caused by the update that disabled Payment and delivery plugins.

> Before the update, the order was not placed.

Was there an error message displayed?


> When the page is refreshed, the number of product orders increases by 1.

> When you click on the order button in the screenshot, nothing happens.

After the order is placed the cart is emptied and this should not work, I will test the checkout process fix the bugs and release a new update, thanks for the bug report.

**Reply by puzkos** [reply]
Hi :)

I can write all bugs, that i will find. As a tester)

1. All Users see here fix name)) 
<img width="528" alt="image" src="https://github.com/givanz/Vvveb/assets/97404154/4c3acf20-86f5-4f0d-9bb1-60c31ce242f8">

2. Just USA and UK have regions. If the region is not in the database, make the button different.
So that the user can manually enter the required region.
<img width="1006" alt="image" src="https://github.com/givanz/Vvveb/assets/97404154/e9f7163e-7329-491c-b5c3-f5771dca17dc">

3. When I select UK and region and press the order button, the page is updated and the following error appears for half a second
![image](https://github.com/givanz/Vvveb/assets/97404154/983ecb32-7709-4e77-b352-b229905a039e)
The screenshot was taken while the page was loading. if this is not stopped, then the page is completely reloaded and empty fields appear.
The screenshot was taken while the page was loading. if this is not stopped, then the page is completely reloaded and empty fields appear.
I tried 10 times to catch this screen)))
Naturally, a new order is not created ...
<img width="750" alt="image" src="https://github.com/givanz/Vvveb/assets/97404154/af149051-3549-4388-8640-81c8c60b4359">

4. if you disable 1 of the payment / delivery methods - it is still displayed when placing an order
but when it is selected, an error is displayed
<img width="740" alt="image" src="https://github.com/givanz/Vvveb/assets/97404154/e852caac-1bdc-4027-b3ff-43753c1570fc">



I can continue like this. Or paint in more detail. How will it be convenient?
I hope this looks appropriate. this is not a list of claims, it will just be easier to communicate with numbers :)) I want to help, because I believe in your project! And I would love to use one!

Best,
Serhii.

**Reply by puzkos** [reply]
5. Why is there a section with products by default
and additionally there is an online store module?
I use the default one. Why is there a section with products by default
and additionally there is an online store module?
I'm using the default one/ however the plugin doesn't work correctly either.

<img width="1902" alt="image" src="https://github.com/givanz/Vvveb/assets/97404154/2a0a9225-87d8-448a-acdf-9b48259ba59e">

6. ошибка списка ордеров в личном кабинете пользователя... 
![image](https://github.com/givanz/Vvveb/assets/97404154/fee3b95f-5e3b-4782-9f79-897403612cec)


7. Just HTML. Nothing happens when you click...
![image](https://github.com/givanz/Vvveb/assets/97404154/d30b25e0-b0d6-4ede-9c8c-81d2d87a5b27)

**Reply by puzkos** [reply]
7. Solution 

```
<head>
<link href="https://cdnjs.cloudflare.com/ajax/libs/lightbox2/2.11.3/css/lightbox.min.css" rel="stylesheet">
</head>

befor close body
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/lightbox2/2.11.3/js/lightbox-plus-jquery.min.js"></script>
</body>

in code
<a href="/public/media/CupStar.png" data-lightbox="image-1" class="d-block mb-4 h-100">
  <img class="img-fluid img-thumbnail" src="/public/media/CupStar.png" alt="">
</a>
```

**Comment by givanz** [comment]
Thanks for the bug reports, I fixed them, some of them require theme update also but theme update feature is not finished you will need to overwrite theme files manually from the zip, just overwrite public/themes/landing.

There were many bugs on checkout for logged in users because the latests tests were done only with guest checkout.

> All Users see here fix name))

Fixed

> Just USA and UK have regions. If the region is not in the database, make the button different. So that the user can manually enter the required region.

Only USA and UK are enabled for demo purposes, to show other countries and zones you need to enabled them from admin > settings > localization > geo location.

Regions for most countries are included and there is no need for manual input from the user, if a region is missing it can be added from admin, the user can enter only city manually.


> When I select UK and region and press the order button, the page is updated and the following error appears for half a second

Fixed

> if you disable 1 of the payment / delivery methods - it is still displayed when placing an order but when it is selected, an error is displayed

This is caused by cache, cache invalidation is not called in some places like plugin deactivation and for some actions manual clear is needed.


> Why is there a section with products by default and additionally there is an online store module? I use the default one. Why is there a section with products by default and additionally there is an online store module?
> I'm using the default one/ however the plugin doesn't work correctly either.

The plugins market plugin adds custom products with product type "plugin" and also the "plugins menu" entry, this plugin is used to display the plugins list on the plugins marketplace https://plugins.vvveb.com/ and is included only for testing.

Currently all plugins are included for easier testing but they will be removed from the zip and they will be available only on the plugins marketplace.

> ошибка списка ордеров в личном кабинете пользователя...

Fixed

> Just HTML. Nothing happens when you click...

You need to enable link and enter a page url 

![Screenshot_2023-07-25_23-46-23](https://github.com/givanz/Vvveb/assets/302738/50685171-54e4-4287-9177-3bde1c606ea8)

> <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script> <script src="https://cdnjs.cloudflare.com/ajax/libs/lightbox2/2.11.3/js/lightbox-plus-jquery.min.js"></script> 


This can be added to the gallery component html code but there should be a better solution without cdn and with more options, the gallery component has basic functionality that will be expanded later.

**Comment by puzkos** [comment]
> Only USA and UK are enabled for demo purposes, to show other countries and zones you need to enabled them from admin > settings > localization > geo location.

Best! I advise you to make the button "Enable all" when you select a specific country.
I turned everything on myself through the database))) It is very inconvenient to poke each region manually (((
The filter is not working. and there are only 3 countries
![image](https://github.com/givanz/Vvveb/assets/97404154/508fa60e-cc9e-4f46-b2b0-9a5f9fa7a21d)

About table. 
https://adminlte.io/themes/v3/iframe.html#
I would suggest using the script from this template (or similar). You can download this template and view the source code.
The table is much more convenient, interactive, with filters.

**Comment by puzkos** [comment]
hi
When updating to 0.0.4, all modules were disabled again))

**Comment by givanz** [comment]
Hi

> Best! I advise you to make the button "Enable all" when you select a specific country.

Bulk actions will be added later for all listings, currently only delete on posts/products have bulk actions.

> The filter is not working. and there are only 3 countries

Filter for country should work after update.

> About table.
> https://adminlte.io/themes/v3/iframe.html#
> I would suggest using the script from this template (or similar). You can download this template and view the source code.
> The table is much more convenient, interactive, with filters

Filters and sorting will be added to all listings, currently only posts, products and countries/zones have filters. 

> When updating to 0.0.4, all modules were disabled again))

The update code that skips the plugins configuration is in the update itself, the update was done with the old update code.

This should work from now on future updates.

**Reply by puzkos** [reply]
![image](https://github.com/givanz/Vvveb/assets/97404154/7298c8aa-4848-4fcc-823c-2809f9c1ab66)
I fixed it, but just as a note
need to make size limit to pictures of product. rounding to an integer always works

<img width="1739" alt="image" src="https://github.com/givanz/Vvveb/assets/97404154/7c5b6d91-8981-49f0-90ec-37b224a6d9b9">
dont work with small amount. rounding to an integer always works

**Comment by givanz** [comment]
> dont work with small amount. rounding to an integer always works

Thanks for the bug report, I fixed product price to work with decimals.

> I fixed it, but just as a note need to make size limit to pictures of product. rounding to an integer always works

I added image resize for product and post images in the last update, you can configure the size from admin > settings > site > media settings 

![Screenshot 2023-07-30 at 20-41-00 Vvveb - Site](https://github.com/givanz/Vvveb/assets/302738/530031a4-2082-43cb-b6b5-234f392e2054)

**Comment by puzkos** [comment]
Hi!

Price is perfect! Also the oareds is working now! You are best! :))


About size of photo... it`s crushed
![image](https://github.com/givanz/Vvveb/assets/97404154/4402621b-1593-435e-a25d-68228d1e1388)



Page editor is not working on other pages. just on main

**Comment by givanz** [comment]
> About size of photo... it`s crushed

Can you please paste the info from `admin > tools > system info` either use copy to clipboard or screenshot.

This is to check if php is using imagick or gd to resize images.

Can you also check if there are errors related to images in `admin > tools > system > error log` and paste them also?

> Page editor is not working on other pages. just on main

Can you please provide more details? The editor is opened from the top `Design page` button or from Edit website > file manager? 

![Screenshot 2023-07-31 at 20-14-40 Powerful and easy to use software to build professional websites blogs or ecommerce stores](https://github.com/givanz/Vvveb/assets/302738/ada54317-3767-4a87-b8c8-74a094ef142c)

![Screenshot 2023-07-31 at 20-17-20 Vvveb - Editor](https://github.com/givanz/Vvveb/assets/302738/5ae40237-6ccf-40ad-83f2-d6949d9465dd)

Neither static pages like contact us nor products or post pages are saved? 

Is there an error message displayed on saving or the successful file saved?  

![Screenshot 2023-07-31 at 20-21-41 Vvveb - Editor](https://github.com/givanz/Vvveb/assets/302738/ebf60079-e309-41ea-8d58-5870ba4230ff)

**Comment by puzkos** [comment]
```
General
Vvveb version	0.0.5
Admin path	/admin/
PHP version	8.1.2-1ubuntu2.13 | apache2handler
Server	Apache/2.4.52 (Ubuntu)
OS version	Linux vps-43475 5.15.0-78-generic #85-Ubuntu SMP Fri Jul 7 15:25:09 UTC 2023 x86_64
Database driver & version	mysqli | Client library version: mysqlnd 8.1.2-1ubuntu2.13 | Server version: 8.0.33-0ubuntu0.22.04.2
PHP time limit	120
PHP memory limit	128M
Max input time	60
Upload max filesize	100M
PHP post max size	100M
Extensions	Core date libxml openssl pcre zlib filter hash json Reflection SPL session standard sodium apache2handler mysqlnd PDO xml calendar ctype curl dom mbstring FFI fileinfo ftp gd gettext iconv exif mysqli pdo_mysql Phar posix readline shmop SimpleXML sockets sysvmsg sysvsem sysvshm tokenizer xmlreader xmlrpc xmlwriter xsl zip Zend OPcache
Page cache	disabled
Debug	enabled
Sql changes check	enabled
Server
Document root	/var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua
Https	on
Http host	story.cupcorn.com.ua
Http x real ip	86.16.134.161
Http x forwarded for	86.16.134.161
Http x forwarded port	443
Http x forwarded proto	https
Http connection	close
Http sec ch ua	"Not/A)Brand";v="99", "Google Chrome";v="115", "Chromium";v="115"
Http sec ch ua mobile	?0
Http sec ch ua platform	"Windows"
Http upgrade insecure requests	1
Http user agent	Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36
Http accept	text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.7
Http sec fetch site	same-origin
Http sec fetch mode	navigate
Http sec fetch user	?1
Http sec fetch dest	document
Http referer	https://story.cupcorn.com.ua/admin/?module=editor/editor&url=/&template=index.html
Http accept encoding	gzip, deflate, br
Http accept language	ru-RU,ru;q=0.9,en-GB;q=0.8,en;q=0.7,en-US;q=0.6,uk;q=0.5
Http cookie	cupcorncomua=u9oo790e5g1l611p6ugggefpvc; _gid=GA1.3.2006727950.1690759353; sidebar-size=; _ga_9SY3V8ETNR=GS1.1.1690812101.4.1.1690812898.0.0.0; _ga=GA1.3.1282354419.1690759352
Path	/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin
Server signature	<address>Apache/2.4.52 (Ubuntu) Server at story.cupcorn.com.ua Port 443</address>
Server software	Apache/2.4.52 (Ubuntu)
Server name	story.cupcorn.com.ua
Server addr	127.0.0.1
Server port	443
Remote addr	86.16.134.161
Request scheme	https
Context prefix	
Context document root	/var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua
Server admin	admin@story.cupcorn.com.ua
Script filename	/var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/admin/index.php
Remote port	59562
Gateway interface	CGI/1.1
Server protocol	HTTP/1.0
Request method	GET
Query string	module=tools/systeminfo
Request uri	/admin/?module=tools/systeminfo
Script name	/admin/index.php
Php self	/admin/index.php
Request time float	1690825732.377
Request time	1690825732
```



Yes! I have errors...
```
[31-Jul-2023 16:01:04 Etc/UTC] PHP Warning:  imagepng(): gd-png: fatal libpng error: bad parameters to zlib
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:04 Etc/UTC] PHP Warning:  imagepng(): gd-png error: setjmp returns error condition
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:04 Etc/UTC] PHP Warning:  imagepng(): gd-png: fatal libpng error: bad parameters to zlib
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:04 Etc/UTC] PHP Warning:  imagepng(): gd-png error: setjmp returns error condition
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:04 Etc/UTC] PHP Warning:  imagepng(): gd-png: fatal libpng error: bad parameters to zlib
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:04 Etc/UTC] PHP Warning:  imagepng(): gd-png error: setjmp returns error condition
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:04 Etc/UTC] PHP Warning:  imagepng(): gd-png: fatal libpng error: bad parameters to zlib
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:04 Etc/UTC] PHP Warning:  imagepng(): gd-png error: setjmp returns error condition
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
0:46 Etc/UTC] PHP Warning:  imagepng(): gd-png error: setjmp returns error condition
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:00:46 Etc/UTC] PHP Warning:  imagepng(): gd-png: fatal libpng error: bad parameters to zlib
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:00:46 Etc/UTC] PHP Warning:  imagepng(): gd-png error: setjmp returns error condition
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:00:46 Etc/UTC] PHP Warning:  imagepng(): gd-png: fatal libpng error: bad parameters to zlib
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:00:46 Etc/UTC] PHP Warning:  imagepng(): gd-png error: setjmp returns error condition
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:00:47 Etc/UTC] PHP Warning:  imagepng(): gd-png: fatal libpng error: bad parameters to zlib
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:00:47 Etc/UTC] PHP Warning:  imagepng(): gd-png error: setjmp returns error condition
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:03 Etc/UTC] PHP Warning:  imagepng(): gd-png: fatal libpng error: bad parameters to zlib
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:03 Etc/UTC] PHP Warning:  imagepng(): gd-png error: setjmp returns error condition
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:03 Etc/UTC] PHP Warning:  imagepng(): gd-png: fatal libpng error: bad parameters to zlib
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:03 Etc/UTC] PHP Warning:  imagepng(): gd-png error: setjmp returns error condition
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:03 Etc/UTC] PHP Warning:  imagepng(): gd-png: fatal libpng error: bad parameters to zlib
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:03 Etc/UTC] PHP Warning:  imagepng(): gd-png error: setjmp returns error condition
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:04 Etc/UTC] PHP Warning:  imagepng(): gd-png: fatal libpng error: bad parameters to zlib
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:04 Etc/UTC] PHP Warning:  imagepng(): gd-png error: setjmp returns error condition
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:04 Etc/UTC] PHP Warning:  imagepng(): gd-png: fatal libpng error: bad parameters to zlib
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:04 Etc/UTC] PHP Warning:  imagepng(): gd-png error: setjmp returns error condition
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:04 Etc/UTC] PHP Warning:  imagepng(): gd-png: fatal libpng error: bad parameters to zlib
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:04 Etc/UTC] PHP Warning:  imagepng(): gd-png error: setjmp returns error condition
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:04 Etc/UTC] PHP Warning:  imagepng(): gd-png: fatal libpng error: bad parameters to zlib
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:04 Etc/UTC] PHP Warning:  imagepng(): gd-png error: setjmp returns error condition
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
0:46 Etc/UTC] PHP Warning:  imagepng(): gd-png error: setjmp returns error condition
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:00:46 Etc/UTC] PHP Warning:  imagepng(): gd-png: fatal libpng error: bad parameters to zlib
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:00:46 Etc/UTC] PHP Warning:  imagepng(): gd-png error: setjmp returns error condition
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:00:46 Etc/UTC] PHP Warning:  imagepng(): gd-png: fatal libpng error: bad parameters to zlib
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:00:46 Etc/UTC] PHP Warning:  imagepng(): gd-png error: setjmp returns error condition
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:00:47 Etc/UTC] PHP Warning:  imagepng(): gd-png: fatal libpng error: bad parameters to zlib
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:00:47 Etc/UTC] PHP Warning:  imagepng(): gd-png error: setjmp returns error condition
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:03 Etc/UTC] PHP Warning:  imagepng(): gd-png: fatal libpng error: bad parameters to zlib
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:03 Etc/UTC] PHP Warning:  imagepng(): gd-png error: setjmp returns error condition
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:03 Etc/UTC] PHP Warning:  imagepng(): gd-png: fatal libpng error: bad parameters to zlib
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:03 Etc/UTC] PHP Warning:  imagepng(): gd-png error: setjmp returns error condition
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:03 Etc/UTC] PHP Warning:  imagepng(): gd-png: fatal libpng error: bad parameters to zlib
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:03 Etc/UTC] PHP Warning:  imagepng(): gd-png error: setjmp returns error condition
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:04 Etc/UTC] PHP Warning:  imagepng(): gd-png: fatal libpng error: bad parameters to zlib
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:04 Etc/UTC] PHP Warning:  imagepng(): gd-png error: setjmp returns error condition
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:04 Etc/UTC] PHP Warning:  imagepng(): gd-png: fatal libpng error: bad parameters to zlib
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:04 Etc/UTC] PHP Warning:  imagepng(): gd-png error: setjmp returns error condition
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:04 Etc/UTC] PHP Warning:  imagepng(): gd-png: fatal libpng error: bad parameters to zlib
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:04 Etc/UTC] PHP Warning:  imagepng(): gd-png error: setjmp returns error condition
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:04 Etc/UTC] PHP Warning:  imagepng(): gd-png: fatal libpng error: bad parameters to zlib
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
[31-Jul-2023 16:01:04 Etc/UTC] PHP Warning:  imagepng(): gd-png error: setjmp returns error condition
 in /var/www/test_cupcorn_usr/data/www/story.cupcorn.com.ua/system/media/gd.php on line 95
```

other page designer not working in admin panel
admin-page-page-edit-open in page editor.

In other cases, it works fine.

**Comment by puzkos** [comment]
also how to remove the search icon? it still doesn't work...

**Comment by givanz** [comment]
Thanks for the error log, I fixed the image resize problem.

> also how to remove the search icon? it still doesn't work...

Search was working only for posts, I added search for products also https://demo.vvveb.com/search/product

You will need to manually update search template /public/themes/landing/search/index.html

> other page designer not working in admin panel admin-page-page-edit-open in page editor.

It would be helpful if you can provide more info regarding saving not working for other pages like some error message.

I'm unable to reproduce the error, sometimes this was caused by wrong file permission on theme html files.
